render/src/bitmap.rs

```diff
@@ -1,14 +1,21 @@
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
 pub struct BitmapHandle(pub usize);
 
-/// Info returned by the `register_bitmap` methods.
-#[derive(Copy, Clone, Debug)]
+#[derive(Clone, Debug)]
 pub struct BitmapInfo {
     pub handle: BitmapHandle,
     pub width: u16,
     pub height: u16,
 }
 
+/// Info returned by the `register_bitmap` methods.
+#[derive(Clone, Debug)]
+pub struct RegisteredBitmapInfo {
```

Comment:
How is this different from simply `ruffle_render::bitmap::Bitmap`? Maybe it can be used instead.

Reply:
This has a `BitmapHandle` (through `BitmapInfo`), which `Bitmap` lacks.

Reply:
However, it turns out that this is no longer necessary after https://github.com/ruffle-rs/ruffle/pull/8316